### PR TITLE
Fixing page titles

### PR DESCRIPTION
**Pull request by mfrachet**
### What does it do?

Adding missing titles on pages

### How to test it?

- Go to /admin/settings/application-infos and see "Settings - Application" in title
- Go to /admin/settings/media-library and see "Settings - Media Library" in the title
- Go to /admin/settings/users-permissions/roles and see  "Settings - Roles" in the title
- Go to /admin/settings/users-permissions/email-templates and see "Settings - Email Templates" in the title

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/9359

### CODE REVIEW

File: packages/strapi-plugin-upload/admin/src/containers/SettingsPage/index.js [hunk 1]
```diff
@@ -111,6 +112,8 @@ const SettingsPage = () => {
 
   return (
     <>
+      <Helmet title="Settings - Media Library" />
```
Comment (HichamELBSI) [inline packages/strapi-plugin-upload/admin/src/containers/SettingsPage/index.js:115]: Don't we need to use translated messages for page title?

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-plugin-upload/admin/src/containers/SettingsPage/index.js (hunk 1 above)
Comment (mfrachet) [reply packages/strapi-plugin-upload/admin/src/containers/SettingsPage/index.js:115]: ooh gosh! totally forgot 🤦🏻 . Thanks !

Comment (HichamELBSI) [reply packages/strapi-plugin-upload/admin/src/containers/SettingsPage/index.js]: 😂

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Pull request merged**